src/main/java/msg/team1/Hi/domain/homebase/presentation/controller/student/StudentHomeBaseController.java

```diff
@@ -58,4 +56,10 @@ public ResponseEntity<Void> updateReservationMembers(@PathVariable("reservation_
         updateReservationMemberService.execute(request, reservationId);
         return ResponseEntity.noContent().build();
     }
+
+    @DeleteMapping("/{reservation_id}/delete")
```

Comment:
@DeleteMapping("/{reservation_id}")이런식으로 변경하는게 좋아보여요. url에는 CRUD 함수의 이름을 사용하지 않는게 RESTful 해보입니다.
```suggestion
    @DeleteMapping("/{reservation_id}")
```